site/gatsby-site/deploy-netlify.toml

```diff
@@ -5,10 +5,10 @@
 [[plugins]]
   package = "@netlify/plugin-gatsby"
 
-[[plugins]]
+[[context.production.plugins]]
```

Comment:
This plugin was running in deploy previews, which doesn't make sense because we use the staging environment for them.

And yes, staging is considered production.